documentation/modules/ROOT/pages/integrations/tracing.adoc

```diff
@@ -0,0 +1,101 @@
+[id="distributed-tracing"]
+= Distributed Tracing
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+== Overview
+
+Observability is an important aspect of microservice-oriented applications.
+One of the key ingeredients for observability is https://microservices.io/patterns/observability/distributed-tracing.html[distributed tracing].
+
+It is necessary to provide additional precautions when an application writes a record to a database that is later processed by Debezium.
+The active trace is effectively demarcated by the write to the database.
+If we want Debezium to join the larger scope application tracer we need to pass the trace metadata to Debezium.
+
+The tracing support is added to Debezium through https://opentracing.io/[OpenTracing] specification.
+It is also necessary to provide a client implementing the specification.
+Debezium was tested with https://www.jaegertracing.io/[Jaeger] implementation.
+
+[WARNING]
+====
+Neither specification JAR files nor Jaeger client are part of the Debezium Connect image.
+The user either needs to extend the image with them or can use the https://strimzi.io/[Strimzi] Kafka image.
+In that case, the tracing of Kafka producer and consumer is also available.
+
+See compose file of https://github.com/debezium/debezium-examples/tree/master/outbox[Outbox example].
+====
+
+== ActivateTracingSpan SMT
+
+The main implementation point of tracing in Debezium is `ActivateTracingSpan` SMT.
+In this case, the application writing to a database is responsible for providing tracing span context.
+The writer should have span context inject into `java.util.Properties` instance that is serialized then serialized and written to the database as a distinct field of the table.
```

Comment:
I don't understand this one.

Reply:
@gunnarmorling Rewritten and ready for review